### ISSUE TITLE
how to get the information of  0.1% Fasle Accept Rate about other models

### ISSUE THREAD

**Issue by PPPPPPPeng** [ronny3050/AdvFaces#7]
A score above 0.28 indicates that two face images belong to the same subject(via ArcFace).
But how to get the threshold information of 0.1% Fasle Accept Rate about SphereFace and FaceNet?

**Comment by ronny3050** [comment]
The thresholds are computed as follows:

1. Extract face features for a subset of LFW where only subjects with minimum two face images are kept (total num images should be 9,614 for 1,680 subjects)
2. Compute cosine similarity scores for all possible genuine pairs (495K pairs) and randomly sample 1M impostor pairs
3. Compute ROC curve via `sklearn.metric.roc_auc_score` and retrieve threshold at FAR=0.1%

Please re-open if you face any issues.
